seeds/data/json/action.js

```diff
@@ -0,0 +1,326 @@
+const DefaultActions = [
```

Comment:
Any reason we don't just want this to be a JSON file?
JSON files can be required directly and just become an object.